src/main/kotlin/com/openlattice/shuttle/MissionControl.kt

```diff
@@ -298,6 +300,8 @@ class MissionControl(
         if (createEntitySets) {
             createMissingEntitySets(flightPlan, contacts)
         }
+
+        principalApi.syncCallingUser()
```

Comment:
This won't execute if the server arg is specified, yeah? (otherwise shuttle server will break)

Reply:
Sounds from @pckeyes like server is skipping missioncontrol?